Update finished portfolio to fix errors for deployment

rylan-marianchuk opened the pull request:
I have completed my portfolio. It is a lot of code for one commit, this is because I already had developed most of the content last year. However the new files I modified this week are 'projects.html' and its corresponding styles at 'styles/projects.css' where I updated the list view for project titles. For the code review, you could ignore all other files.

imadkar commented on portfolio/src/main/webapp/index.html:
```diff
@@ -1,18 +1,36 @@
+    </div>
+
+    <ul class="side-menu">
+        <li class="menu-items"><a href="#"> H O M E </a></li>
```
I'd recommend creating a css style using the letter-spacing [property](https://developer.mozilla.org/en-US/docs/Web/CSS/letter-spacing). This way you can keep the url as "Home" which is more semantic and allows crawlers and screenreaders to work better!

imadkar commented on portfolio/src/main/webapp/index.html:
```diff
@@ -1,18 +1,36 @@
+        <li class="menu-items"><a href="projects.html">P R O J E C T S</a></li>
+        <li class="menu-items"><a href="personal2.html">B I O</a></li>
+    </ul>
+    <div class="side-nav"></div>
```
Maybe add a comment here and the other pages on why this is an empty div?

imadkar commented on portfolio/src/main/webapp/index.html:
```diff
@@ -1,18 +1,36 @@
+    <div class="top-nav">
+        <button class="trigger" style="margin-top: 0;"><img src="img/MenuIcon.png" style="max-height: 100%; max-width: 100%; width: 50%; height: 35%; pointer-events: none;"></button>
+        <div class="navbar">
+            <a href="#">R Y L A N  &nbsp; M A R I A N C H U K</a>
```
I'd recommend creating a style using [property](https://developer.mozilla.org/en-US/docs/Web/CSS/letter-spacing) to create your spaced letters effect here and the other pages. This way when a crawler for a search engine goes through your site, it'll parse your name correctly so the page will show up in search results for your name!

imadkar commented on portfolio/src/main/webapp/index.html:
```diff
@@ -1,18 +1,36 @@
+</head>
+<body>
+    <div class="top-nav">
+        <button class="trigger" style="margin-top: 0;"><img src="img/MenuIcon.png" style="max-height: 100%; max-width: 100%; width: 50%; height: 35%; pointer-events: none;"></button>
```
I'd recommend pulling the styles into a css class. This way if you make a change you only need to modify the css class and not each page.

imadkar commented on portfolio/src/main/webapp/index.html:
```diff
@@ -1,18 +1,36 @@
+    </ul>
+    <div class="side-nav"></div>
+
+
```
nit: this is an extra line

imadkar commented on portfolio/src/main/webapp/personal2.html:
```diff
@@ -0,0 +1,87 @@
+
+            <img src="https://gnrbla.bn.files.1drv.com/y4mGJHhoylOPoqmnunCy3C4FSCuXpXCeNgV6gyl6Zt26XOQXkDlsxuutCHNs--GDH0_elnPesusjYCgPLsivGxSsbcD-r3F_GN_5c6D_P1jZv5T7d6HLu-VdMzPrtTkDR1YdAhCCzQAtmLhtDvfk0iPhGywOqZLRbZU1RGsITyVbro_r91AkseaHlESu241FAycu24a_5bQx5YPElpuabMOyA?width=2848&height=4288&cropmode=none" class="img-fluid mb-4 rellax img_personal" alt="" data-rellax-speed="2" style="margin-to ...
+
+            <!--  <img src="img/profile/2017-06-02%2010.18.07%201.jpg" class="img-fluid mb-4 rellax img_personal" alt="" data-rellax-speed="1" style="margin-top: 8vh;"> -->
```
Did you mean to remove this?

imadkar commented on portfolio/src/main/webapp/personal2.html:
```diff
@@ -0,0 +1,87 @@
+            <!--  <img src="img/profile/2017-06-02%2010.18.07%201.jpg" class="img-fluid mb-4 rellax img_personal" alt="" data-rellax-speed="1" style="margin-top: 8vh;"> -->
+
+        </div>
+        <!--Grid column-->
```
Were you going to add another column or should this be removed?

imadkar commented on portfolio/src/main/webapp/styles/projects.css:
```diff
@@ -0,0 +1,169 @@
+    background-color: rgba(135, 135, 135, 0.8);
+
+}
+
```
nit: Could you remove this extra line?

rylan-marianchuk commented:
Thanks for the review and suggestions Imad. It would be a good idea to not have my name puncuated with spaces throughout to have my name come up in search. Going forward I will not give you large amounts of code to review like this. I have committed (as the tutorial rerquired) a single change in the java Servlet file.

rylan-marianchuk commented:
I have implemented a minimal functioning comment system on my index.html page. I have sectioned off the work into 5 steps as the tutorial has asked. The final commit I added css styling and query for permanent data storage.
